ethereum-types/src/hash.rs

```diff
@@ -6,47 +6,24 @@ use serde::{Serialize, Serializer, Deserialize, Deserializer};
 #[cfg(feature="serialize")]
```

Comment:
`serialize` feature contains deserialization implementations. What about to call this feature `serde`? It looks more consistent with other crates, like: `chrono`, `url`, etc., also there may be a new serialization crates in the future.

Reply:
The issue was discussed here: https://github.com/paritytech/primitives/pull/10#discussion_r163241910 Seems it's not possible.